AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_component_validation.py

```diff
@@ -22,12 +22,13 @@ def compare_vec3(expected: Math.Vector3, actual: Math.Vector3) -> bool:
 
     return: boolean result of whether or not the two vector3s are the same.
     """
-    x_is_close = Math.Math_IsClose(expected.get_property('x'), actual.get_property('x'), 0.001)
-    y_is_close = Math.Math_IsClose(expected.get_property('y'), actual.get_property('y'), 0.001)
-    z_is_close = Math.Math_IsClose(expected.get_property('z'), actual.get_property('z'), 0.001)
+    x_is_close = Math.Math_IsClose(expected.get_property('x'), actual.get_property('x'), 0.01)
+    y_is_close = Math.Math_IsClose(expected.get_property('y'), actual.get_property('y'), 0.01)
+    z_is_close = Math.Math_IsClose(expected.get_property('z'), actual.get_property('z'), 0.01)
```

Comment:
suggestion: if `0.01` is repeated in many places it'd might be better to put it in a global variable.